### PR TITLE
Delete assessments

### PR DESCRIPTION
**Pull request by dspencer001**

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -50,3 +50,4 @@ rules:
     }
   }]
   jsx-a11y/no-static-element-interactions: 0
+  react/require-default-props: off
```
Comment (BenHSmith13) [inline .eslintrc.yml]: I don't think off is the right. I think you have to set it to 0 or something

Comment (dspencer001) [inline .eslintrc.yml]: They use both [here](http://eslint.org/docs/user-guide/configuring)

Comment (BenHSmith13) [inline .eslintrc.yml]: Fair enough

**Comment by BenHSmith13** [comment]
I think this looks good.

**Pull request merged**